proto/ibc/core/channel/v1/channel.proto

```diff
@@ -112,6 +112,8 @@ message Packet {
       [(gogoproto.moretags) = "yaml:\"timeout_height\"", (gogoproto.nullable) = false];
   // block timestamp (in nanoseconds) after which the packet times out
   uint64 timeout_timestamp = 8 [(gogoproto.moretags) = "yaml:\"timeout_timestamp\""];
+  // delay period before which packet cannot be verified on receiving chain
```

Comment:
I think this is also on the sending chain in the case of acknowledgements. Maybe the comment should be updated to reflect the idea that it is needed for any sort of packet verification?